### ISSUE TITLE
Cannot name NDI output stream

### ISSUE THREAD

**Issue by llamafilm** [CESNET/UltraGrid#163]
The help text says I should be able to name the NDI stream but when I try this it says "unknown option".  It works fine if I omit the name, then it uses a default name of "uv".

```
 % ./bin/uv -t testcard:1920:1080:24:UYVY -d ndi:name=ultragrid
UltraGrid 1.6+ (master rev cbb6a20c built Jun  4 2021 11:28:54)

Display device   : ndi
Capture device   : testcard
Audio capture    : none
Audio playback   : none
MTU              : 9000 B
Video compression: none
Audio codec      : PCM
Network protocol : UltraGrid RTP
Audio FEC        : none
Video FEC        : none

[NAT] Private outbound IPv4 address detected and bound to a non-dynamic port. Consider adding '-N' option for NAT traversal.
[NDI disp.] Unknown option: name=ultragrid!
Unable to open display device: ndi
Exit
```

**Comment by MartinPulec** [comment]
Thanks, this was an copy&paste error, it should be fixed with e6a59fe0.

**Comment by llamafilm** [comment]
Works for me, thanks!
